nodeadm/internal/kubelet/kubeconfig.go

```diff
@@ -68,10 +71,3 @@ func generateKubeconfig(cfg *api.NodeConfig) ([]byte, error) {
 	}
 	return buf.Bytes(), nil
 }
-
```

Comment:
might worth move this to util is we have future changes that need to write files

Reply:
the extra calls are not too noisy, but it is pretty common so i can add a wrapper/helper